core/src/main/java/org/bitcoinj/evolution/QuorumState.java

```diff
@@ -121,19 +121,12 @@ public void applyDiff(Peer peer, DualBlockChain blockChain,
         StoredBlock block;
         boolean isSyncingHeadersFirst = context.peerGroup.getSyncStage() == PeerGroup.SyncStage.MNLIST;
 
-        long newHeight = ((CoinbaseTx) mnlistdiff.coinBaseTx.getExtraPayloadObject()).getHeight();
-        block = blockChain.getBlock(mnlistdiff.getBlockHash());
-
-        if(!isLoadingBootStrap && block.getHeight() != newHeight)
-            throw new ProtocolException("mnlistdiff blockhash (height="+block.getHeight()+" doesn't match coinbase blockheight: " + newHeight);
-
```

Comment:
```java
        block = blockChain.getBlock(mnlistdiff.getBlockHash());
```
Based on the logs, this line of code was probably taking too long.  We don't really need it because if the block heights don't match, then neither will the block hashes which will be discovered later in the code.